airbyte-integrations/connectors/source-smartsheets/Dockerfile

```diff
@@ -6,13 +6,13 @@ RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 ENV CODE_PATH="source_smartsheets"
 
 WORKDIR /airbyte/integration_code
-COPY $CODE_PATH ./$CODE_PATH
-COPY main.py ./
 COPY setup.py ./
 RUN pip install .
+COPY main.py ./
+COPY $CODE_PATH ./$CODE_PATH
```

Comment:
Why this lines moved ?

Reply:
`COPY`s should be done after `pip install` to speedup builds
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache

```For the ADD and COPY instructions, the contents of the file(s) in the image are examined and a checksum is calculated for each file. The last-modified and last-accessed times of the file(s) are not considered in these checksums. During the cache lookup, the checksum is compared against the checksum in the existing images. If anything has changed in the file(s), such as the contents and metadata, then the cache is invalidated.```

```Once the cache is invalidated, all subsequent Dockerfile commands generate new images and the cache is not used.```